src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --nav-color: #00acdc;
+  --link-height: 60px;
+}
+
+* {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  max-width: 100vw;
+}
+
+.navbar {
+  margin: 0;
+}
+
+.list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+  list-style-type: none;
+}
+
+.list li {
```

Comment:
Avoid using tag names for styling (except body and html).
Check the css part of this [doc](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md).

Reply:
You can use a class like list__item